src/website2/src/views/cleanairforum/partners/PartnersPage.tsx

```
@@ -0,0 +1,148 @@
+    if (!section.pages.includes('partners')) return false;
+    const sectionHTML = renderContent(section.content);
+    return isValidHTMLContent(sectionHTML);
+  });
+
+  return (
+    <div className="px-4 prose max-w-none lg:px-0">
+      {showMainPartners && (
+        <div className="py-4">
+          <h1 className="text-2xl font-bold">Partners</h1>
+          <div
+            dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)